Translation cleanup

Sicaine opened the pull request:
Cleanup of two textes. Introduced myself to poedit, analysed db/update.php which is probably not used anymore

Commit:
cleanup gettext po file through poedit

I guess it was mostly managed with poedit and a few lines where updated
manualy.

Clean state to work with poedit.

Commit:
fix translation text in registration

Commit:
fix translation text for info field and hint to departure

fpletz commented on locale/de_DE.UTF-8/LC_MESSAGES/default.po:
```diff
@@ -36,10 +36,11 @@ msgstr ""
 msgstr ""
-"Du bist nicht als verfügbar markiert. Bitte melde Dich beim Infobus, "
-"und bestätige Deine Verfügbarkeit."
+"Du bist nicht als verfügbar markiert. Bitte melde Dich beim Infobus, und "
```
Im Deutschen kommt vor dem und kein Komma.

fpletz commented on includes/pages/guest_login.php:
```diff
@@ -169,7 +169,7 @@ function guest_register() {
   
   return page_with_title(register_title(), array(
-      _("By completing this form you're registering as an Angel. This script will create you an account in the angel task sheduler."),
+      _("By completing this form you're registering as a Chaos-Angel. This script will create you an account in the angel task sheduler."),
```
Bitte keine Chaos-Referenzen.

Commit:
update mo/po file and fix occured errors

Before this commit the translations where changed manualy. poedit found
issues matching used constants in code and translations.

Regenerated/updated po file with poedit including relativ path instead
of full.

fpletz commented:
Looks good. :+1:
